### PR TITLE
Remove review title from entry form

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/1739

### CODE REVIEW

File: src/amo/components/AddonReview.js [hunk 1]
```diff
@@ -22,33 +22,29 @@ export class AddonReviewBase extends React.Component {
...
     }
     this.overlayCard = null;
     this.reviewTextarea = null;
-    this.reviewTitleInput = null;
   }
 
   componentWillReceiveProps(nextProps) {
     const { review } = nextProps;
     if (review) {
       this.setState({
```
Comment (tofumatt) [inline src/amo/components/AddonReview.js]: I bet this could be one line now, if you wanted.

File: src/amo/components/AddonReview.js [hunk 2]
```diff
@@ -104,15 +94,6 @@ export class AddonReviewBase extends React.Component {
         <form onSubmit={this.onSubmit} ref={(ref) => { this.reviewForm = ref; }}>
           <div className="AddonReview-form-input">
             {errorHandler.hasError() ? errorHandler.renderError() : null}
-            <input
```
Comment (tofumatt) [inline src/amo/components/AddonReview.js]: Did this never have a label? Looks like the `<textarea>` is missing one too. I know it's unrelated to this patch, but could you add one?

Comment (kumar303) [inline src/amo/components/AddonReview.js]: The textarea doesn't have a label in the mock. Are you suggesting we should add a hidden one? I'm not too familiar with a11y -- what purpose does that serve?

Comment (tofumatt) [inline src/amo/components/AddonReview.js]: Yeah, a hidden one will give extra context to screenreaders. Something simple like "Your review text" should be fine.

Comment (kumar303) [inline src/amo/components/AddonReview.js]: ok, I added a hidden label

**Comment by kumar303** [comment]
Thanks for the review

**Pull request merged**